### DISCUSSION TITLE
Is this an error

### DISCUSSION THREAD

**Discussion by IHateYourCode** [texstudio-org/texstudio#1809] [Q&A] [answered]
[Tips & Tricks - Auxiliary Files]: https://github.com/texstudio-org/texstudio/wiki/Tips-And-Tricks#using-a-separate-directory-for-auxiliary-files

>Similar for biber, you can change it to `biber.exe --output_directory=aux-files %`.

from [Tips & Tricks - Auxiliary Files], is the `%` meant to be there?

**Comment by sunderme** [comment, marked as answer]
Yes, % is meant to be there.
